doc/classes/EditorUndoRedoManager.xml

```diff
@@ -10,6 +10,8 @@
 		- If the object is a built-in resource, use the scene from its path;
 		- If the object is external resource or anything else, use global history.
 		This guessing can sometimes yield false results, so you can provide a custom context object when creating an action.
+		[EditorUndoRedoManager] is intended to be used for Godot editor plugins. You can obtain it using [method EditorPlugin.get_undo_redo]. For non-editor uses or plugins that don't need to integrate with the editor's undo history, use [UndoRedo] instead.
```

Comment:
```suggestion
		[EditorUndoRedoManager] is intended to be used by Godot editor plugins. You can obtain it using [method EditorPlugin.get_undo_redo]. For non-editor uses or plugins that don't need to integrate with the editor's undo history, use [UndoRedo] instead.
```

Reply:
I think this one is correct. The users are supposed to use it for their plugins, that's what it means.

Reply:
I understand what you mean, but then it should be something like "for making Godot editor plugins". "by" seems like a better options.

Reply:
Could also be "in editor plugins" ;)

Reply:
Too late 🙃